blinkt.go

```diff
@@ -40,6 +47,25 @@ func writeByte(val int) {
 	}
 }
 
+func (bl *Blinkt) InitStopper() {
+
+	signalChan := make(chan os.Signal, 1)
+	signal.Notify(signalChan, os.Interrupt)
+	fmt.Println("Press Control + C to stop")
+
+	go func() {
+		for range signalChan {
+			bl.Clear()
+			bl.Show()
+			os.Exit(1)
+		}
+	}()
+}
+
+func (bl *Blinkt) Sleep(ms int) {
```

Comment:
I originally called it `Delay `(simple lift and shift from the examples), but the dotted import was causing a conflict between `blinkt.Delay` and `rpi.Delay`.  My initial response was to use a different moniker but I've later addressed this by removing the dot import of `rpi `and explicitly referring to `rpi `throughout blinkt.go.  Merging this into the library will also, and for the same reason, break any examples where `Delay` is declared locally.

As I'm sure you've already established, there is a hint of learning by doing - in terms of go - on my behalf here, so I'm not entirely sure whether this was the most appropriate approach, or if indeed there are any alternatives. (I really appreciate your patience)